frontend/src/organizations/components/OrganizationDetails.js

```diff
@@ -63,40 +63,47 @@ const OrganizationDetails = props => {
         {props.organization.organizationAddress &&
         <div className="address">
           <dl className="dl-horizontal">
-            <dt>Address:</dt>
-            <dd>{props.organization.organizationAddress.addressLine1}</dd>
-            <dt />
-            <dd>{props.organization.organizationAddress.addressLine2}</dd>
-            <dt />
-            <dd>{props.organization.organizationAddress.addressLine3}</dd>
-            <dt />
-            <dd>{props.organization.organizationAddress.city && `${props.organization.organizationAddress.city}, `}
+            <dt style={{ width: "300px" }}><strong>Head Office Address:</strong></dt>
+            <dd>{props.organization.organizationAddress.addressLine1 && `${props.organization.organizationAddress.addressLine1}, `}{props.organization.organizationAddress.addressLine2 && `${props.organization.organizationAddress.addressLine2}, `}{props.organization.organizationAddress.addressLine3 && `${props.organization.organizationAddress.addressLine3}, `}{props.organization.organizationAddress.city && `${props.organization.organizationAddress.city}, `}
```

Comment:
can we try and simplify these long variable names and use something similar to the following code:

`const {
  addressLine1,
  addressLine2,
  addressLine3,
  city,
} = props.organization.organizationAddress;

const formatAddress = (address) => address && `${address}, `;

return (
  <>
    <dt style={{ width: "300px" }}><strong>Head Office Address:</strong></dt>
    <dd>
      {formatAddress(addressLine1)}
      {formatAddress(addressLine2)}
      {formatAddress(addressLine3)}
      {formatAddress(city)}
    </dd>
  </>
);`

Reply:
Make sure to pull in release into this branch as well. There are address changes that may be applicable.